server/mailet/mock-smtp-server/pom.xml

```diff
@@ -60,5 +64,20 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.subethamail</groupId>
+            <artifactId>subethasmtp</artifactId>
+            <version>3.1.7</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.james</groupId>
```

Comment:
that dependency should be up with the ${james.groupId} groupId

Reply:
thanks! if there is no more comments, I will fix it and rebase + squash